app/controllers/api/v1/search_controller.rb

```diff
@@ -619,6 +632,29 @@ def self.match_studies_by_facet(query_results, search_facets)
         matches
       end
 
+      # Simplify TDR results to be mappable for the UI badges for facet search
+      def self.simplify_TDR_facet_search_results(query_results, search_facets)
+        simple_TDR_result = {}
+        simple_TDR_results = []
+        query_results.each do |result|
+          unless result[0].nil?
+            result_details = result[1]
+            accession = result_details[:accession]
+            facet_matches = result_details[:facet_matches]
+            simple_TDR_result[:study_accession] = accession
+            if facet_matches.present?
+              facet_matches[0].each_pair { |key, val| 
```

Comment:
Why are we only ever looking at the first entry in `facet_matches`?  Don't we want to iterate over all of them to see what matches?

Also, style nit: when doing a multi-line block in Ruby, don't use the inline form of `{ |foo| }`, but rather the `do` keyword:

```suggestion
              facet_matches[0].each_pair do |key, val| 
```